style.css

```
@@ -0,0 +1,100 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    height: 100%;
+}
+
+body {
+    margin: 0 auto;
+    height: 100%;
```

```suggestion
    height: 100vh;
}

body {
    margin: 0 auto;
    height: 100vh;
```
Usually, those top-level container sizes should be expressed in vh (visual height) or vw (visual width).